src/Core/src/Primitives/HybridWebViewInvokeJavaScriptRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Text.Json.Serialization.Metadata;
+using System.Threading.Tasks;
+
+namespace Microsoft.Maui
+{
+	public class HybridWebViewInvokeJavaScriptRequest(string methodName, object?[]? paramValues, JsonTypeInfo?[]? paramJsonTypeInfos)
+		: TaskCompletionSource<string>
+	{
+		public string MethodName { get; } = methodName;
+		public object?[]? ParamValues { get; } = paramValues;
+		public JsonTypeInfo?[]? ParamJsonTypeInfos { get; } = paramJsonTypeInfos;
```

Comment:
```suggestion
		public object?[]? ParameterValues { get; } = paramValues;
		public JsonTypeInfo?[]? ParameterJsonTypeInfos { get; } = paramJsonTypeInfos;
```

Reply:
This probably should also have the return type infos.

Reply:
If we make the params all strings, then we don't need any json type info in the handlers at all.

Reply:
Added return type info